lib/backend_common/backend_common/__init__.py

```diff
@@ -27,11 +27,27 @@
 logger = cli_common.log.get_logger(__name__)
 
 
-def create_app(name, extensions=[], config=None, redirect_root_to_api=True, **kw):
-
-    logger.debug('Initializing', app=name)
-
-    app = flask.Flask(name, **kw)
+def create_app(
+            project_name,
+            app_name=None,
+            extensions=[],
+            config=None,
+            redirect_root_to_api=True,
+            **kw
+        ):
+    '''
+    Create a new Flask backend application
+    app_name is the Python application name, used as Flask import_name
+    project_name is a "nice" name, used to identify the application
+    '''
+
+    if app_name is None:
+        app_name = project_name.replace('-', '_')
+
+    logger.debug('Initializing', app=app_name)
+
+    app = flask.Flask(import_name=app_name, **kw)
+    app.name = project_name
```

Comment:
I suppose with this change we can actually skip changing all the ``config.py`` files since ``app_name`` will be correctly assumed.